.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+env:
+  - DB=MYSQL CORE_RELEASE=3.1
+
+matrix:
+  include:
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.5
+
+
+before_script:
+  - composer self-update
```

Comment:
It then does it again - you're right - we don't need this in our `before_script` anymore...